clients/drcachesim/CMakeLists.txt

```diff
@@ -793,7 +795,8 @@ if (BUILD_TESTS)
   set_tests_properties(tool.reuse_distance.unit_tests PROPERTIES TIMEOUT ${test_seconds})
 
   add_executable(tool.drcachesim.unit_tests tests/drcachesim_unit_tests.cpp
-    tests/cache_replacement_policy_unit_test.cpp tests/config_reader_unit_test.cpp)
+    tests/cache_replacement_policy_unit_test.cpp tests/config_reader_unit_test.cpp
+    tests/v2p_reader_unit_test.cpp)
```

Comment:
Hmm, this isn't part of dr$sim though.  (Or is there a plan to add dr$sim support for v2p?)

Reply:
The cache simulator can also benefit from this reader, as it uses the same mechanism (inherited by simulator_t) for physical addresses as the tlb simulator.